src/main/java/org/springframework/integration/kafka/support/ProducerMetadata.java

```diff
@@ -49,6 +49,10 @@
 
 	private int batchBytes = 16384;
 
+	private int sendTimeout = 0;
+
+	private SendMode sendMode = SendMode.async;
```

Comment:
I think that this should be a boolean value (`sync`), since there are only two possible modes - also simplifies evaluation.